docs/src/components/FancyLink.tsx

```diff
@@ -0,0 +1,73 @@
+import React from "react";
+import styled from "styled-components";
+import { Figma, Github, ReactJs, Spectrum, Storybook } from "@icons-pack/react-simple-icons";
+import { ColorPicker, Download, FlightDirect, Google, Tips } from "@kiwicom/orbit-components/icons";
+import { Tile } from "@kiwicom/orbit-components";
+
+import useIsUrlExternal from "../hooks/useIsUrlExternal";
+
+interface FancyLinkProps {
+  download: boolean;
```

Comment:
i suggest you to have `download` as optional, because it's better to add it when it's needed, rather than add `false` to prop explicitly ;)

Reply:
Certainly better. Was intended. 🙈